lib/internal/repl.js

```diff
@@ -22,8 +22,8 @@ function createRepl(env, opts, cb) {
   opts = opts || {
     ignoreUndefined: false,
     terminal: process.stdout.isTTY,
-    useGlobal: true,
-    breakEvalOnSigint: true
+    breakEvalOnSigint: true,
```

Comment:
If you switch the order of these lines, the diff would only show the `useGlobal` line changing.